src/core/aria.js

```diff
@@ -0,0 +1,7 @@
+import { sub } from "core/pubsubhub";
+
+document.body.setAttribute("aria-busy", "true");
```

Comment:
Actually, I think this should be aria-hidden.  Some browsers do not allow the entire webview area to be marked as busy.  We could try aria-hidden.  Alternately, we could wrap the entire content in a div and then mark that div as busy / hidden.